### ISSUE TITLE
Missing secondary / accent color

### ISSUE THREAD

**Issue by BearOutThere** [rdlauer/nativescript-theme-builder#9]
Google design guidelines include secondary / accent color that can be used in a color scheme.  However, that color is missing from the theme builder.  (It is included in the sass variables thought).  Would be very helpful to have that color clearly available for customization.

**Comment by rdlauer** [comment]
Yeah, that's a good idea. Any thoughts on how to actually display that color in the simulators? I'd like to have a way to show off the color instead of it just being an option to choose on the left.

**Comment by BearOutThere** [comment]
I would think that the Primary and Secondary(accent) colors would be included in the Global section in the builder's left panel.  It would also be good to include it in the themes drop  down menu for ease of selection.

**Comment by rdlauer** [comment]
I'm just trying to think of how to *display* the accent color in the simulators as well...
